### PR TITLE
update tutor

### PR DESCRIPTION
**Pull request by n0s4**
There have been a couple of inconsistencies pointed out in #2679 & #2590 (the latter seems to have been quiet for a few days but we could still do each separately).

I also added a few more example "solutions" for the demos, some of which aren't exactly necessary since they are pretty obvious but some already have them so it can't hurt to be consistent.

### CODE REVIEW

**Comment by n0s4** [comment]
I've noticed also that "Insert mode" and "Normal mode" are capitalized but "command mode" isn't. I think I'll fix that here as well.

File: runtime/tutor.txt [hunk 1]
```diff
@@ -621,21 +621,21 @@ _________________________________________________________________
  Type J to join together lines in selection.
 
  1. Move the cursor to the line below marked -->.
- 2. Type J thrice or 3J to join the sentence to one line.
+ 2. Type x four times or 4x to join the sentence to one line.
```
Comment (the-mikedavis) [inline runtime/tutor.txt:624]:
```suggestion
 2. Type x four times or 4x to select all four lines.
```

**Review by the-mikedavis — APPROVED** [review]
I like the solution lines, I think it's good to have an idea where you're supposed to end up 👍

**Comment by n0s4** [comment]
Alright, I've resolved them now @the-mikedavis.

**Pull request merged**